src/bytes.rs

```diff
@@ -175,57 +192,85 @@ impl<const BITS: usize, const LIMBS: usize> Uint<BITS, LIMBS> {
         bytes
     }
 
-    /// Creates a new integer from a little endian stream of bytes.
+    /// Converts a big-endian byte array of size exactly
+    /// [`Self::BYTES`] to [`Uint`].
```

Comment:
any reason for reordering methods?

Reply:
consistency

Reply:
ya, just makes the functional review harder